src/components/bundles.json

```diff
@@ -1 +1 @@
/sdk/core/core-auth/README.md"},{"name":"@azure/cosmos","description":"Microsoft Azure Cosmos DB Service Node.js SDK for SQL API","version":"3.17.1","url":"https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/cosmosdb/cosmos/README.md"},{"name":"bcryptjs","description":"Optimized bcrypt in plain JavaScript with zero dependencies. Compatible to 'bcrypt'.","version":"2.4.3"},{"name":"casbin.js","description":"An authorization library that supports access control models like ACL, RBAC, ABAC in Typescript","version":"1.0.1","url":"https://casbin.org"},{"name":"@cfworker/cosmos","description":"Azure Cosmos DB client for Cloudflare Workers and service workers","version":"1.13.4","url":"https://github.com/cfworker/cfworker/tree/master/packages/cosmos/README.md"},{"name":"cheerio","description":"Tiny, fast, and elegant implementation of core jQuery designed specifically for the server","version":"1.0.0-rc.11","url":"https://cheerio.js.org/"},{"name":"faunadb","description":"FaunaDB Javascript driver for Node.JS and Browsers","version":"4.5.4","url":"https://fauna.com"},{"name":"graphql","description":"A Query Language and Runtime which can target any service.","version":"16.3.0","url":"https://github.com/graphql/graphql-js"},{"name":"jose","description":"'JSON Web Almost Everything' - JWA, JWS, JWE, JWT, JWK, JWKS for Node.js, Browser, Cloudflare Workers, Deno, and other Web-interoperable runtimes","version":"4.10.4","url":"https://github.com/panva/jose"},{"name":"mmdynamo","description":"The official C8 Dynamo JavaScript driver.","version":"0.3.0","url":"https://github.com/macrometacorp/mmdynamo"},{"name":"realm-web","description":"Authenticate and communicate with the MongoDB Realm platform, from your web-browser","version":"1.7.0","url":"https://github.com/realm/realm-js/tree/master/packages/realm-web#readme"},{"name":"@sagi.io/workers-jwt","description":"Generate JWTs on Cloudflare Workers using the WebCrypto API","version":"0.0.22","url":"https://sagi.io"},{"name":"@sendgrid/mail","description":"Twilio SendGrid NodeJS mail service","version":"7.7.0","url":"https://sendgrid.com"},{"name":"@sentry/tracing","description":"Extensions for Sentry AM","version":"7.11.1","url":"https://github.com/getsentry/sentry-javascript/tree/master/packages/tracing"},{"name":"stripe","description":"Stripe API wrapper","version":"11.1.0","url":"https://github.com/stripe/stripe-node"},{"name":"@supabase/supabase-js","description":"Isomorphic Javascript client for Supabase","version":"2.12.0","url":"https://github.com/supabase/supabase-js"},{"name":"toucan-js","description":"Cloudflare Workers client for Sentry","version":"2.5.0","url":"https://github.com/robertcepa/toucan-js#readme"},{"name":"@xata.io/client","description":"Xata.io SDK for TypeScript and JavaScript","version":"0.21.1","url":"https://xata.io/docs/sdk/getting-started"},{"name":"@zuplo/sf-js-sdk","description":"Simple Javascript/Typescript module for interacting with Salesforce APIs","version":"0.12.0"}]}
```

Comment:
@ntotten - I'm not sure if this is needed but I saw that you checked this in a few days ago.